### PR TITLE
Header Stamping

### PR DESCRIPTION
**Pull request by naomielst**
_Note_: Before submitting this pull request, please review our [contributing
guidelines](https://docs.celeryproject.org/en/master/contributing.html).

## Description
Implement group stamping - when a group ID is frozen, its ID is stamped on all tasks in the group. 
A task's group stamps are ordered according to the order of freezing.

### CODE REVIEW

File: .github/workflows/python-package.yml [hunk 1]
```diff
@@ -1,74 +0,0 @@
-# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
```
Comment (auvipy) [inline .github/workflows/python-package.yml:1]: I am not sure why the whole CI is deleted?

**Review by auvipy — CHANGES_REQUESTED** [review]
can you please check the build failures?

**Comment by auvipy** [comment]
btw, for future contribution you might follow https://opensource.guide/how-to-contribute/ , and please fork celery and create a branch there and make a PR. that will be better for you IMHO.

File: .github/workflows/python-package.yml (hunk 1 above)
Comment (thedrow) [reply .github/workflows/python-package.yml:1]: It's just the comment. Not the entire file.

**Pull request closed without merging**